connexion/serialization/deserializers.py

```diff
@@ -0,0 +1,123 @@
+import logging
+import re
+
+from jsonschema import ValidationError
+
+from ..exceptions import ExtraParameterProblem
+from ..problem import problem
+from ..types import coerce_type
+from ..utils import is_null
+
+logger = logging.getLogger('connexion.serialization.deserializers')
```

Comment:
shouldn't `__name__` be sufficient here?

Reply:
I'm not sure why, but all of the other getLogger calls in connexion are hard-coded

Reply:
If you don't mind, I might go through everything and clean it up for ya. Not sure when, but perhaps I can find some time today's evening or tomorrow.